addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -4548,6 +4549,37 @@ QUnit.module("Views", (hooks) => {
         assert.strictEqual(target.querySelector(".o_field_widget input").value, "01/25/2017");
     });
 
+    QUnit.test("allow to use compute dates (+5d for instance)", async function (assert) {
```

Comment:
shouldn't this be in date_field_tests.js?

Reply:
nope, the commit fixes the date picker, the test should be in date_picker_tests.js